### ISSUE TITLE
Provide error message when offline

### ISSUE THREAD

**Issue by cassidyjames** [endlessm/gamestar-mechanic-linux#2]
The current experience when launched offline is less than ideal:

![image](https://github.com/endlessm/gamestar-mechanic-linux/assets/611168/e688a6b0-1d7c-4747-a6e3-deb7c93c18d0)

I understand this is loading an external website[^1], so making it run locally is probably out of scope, but we should at least see if we can include a fallback HTML page that just states the app requires the Internet?

This is especially relevant if we plan to include the app in OS images.

[^1]: If not, uh, we've done something terribly wrong where it somehow still requires internet access 😝

**Comment by manuq** [comment]
Yes, I agree. There shouldn't be a need for going online for the "quest" and "workshop" tabs and the initial pages. But since we are packaging an existing app for which we don't have control, an error message makes sense.

**Comment by cassidyjames** [comment]
I threw a quick page together using the assets from gamestarmechanic.com, feel free to use this as a starting point. :) 

![Screenshot 2023-11-13 at 12-22-12 Gamestar Mechanic](https://github.com/endlessm/gamestar-mechanic-linux/assets/611168/3469b89d-dd46-4c51-b8d3-3d435aca6ffc)

[index.zip](https://github.com/endlessm/gamestar-mechanic-linux/files/13340139/index.zip)

It's kind of intentionally stupidly simple because I didn't spend much time on it, plus I didn't know what capabilities the old Chromium would have.
